src/jwt/manager.rs

```diff
@@ -1,21 +1,27 @@
 use crate::errors::ServiceError;
 use crate::jwt::model::Claims;
+use crate::user::model::SlimUser;
 use jsonwebtoken::{decode, encode, Algorithm, Header, Validation};
 
-pub fn create_token(email: &str) -> Result<String, ServiceError> {
-    let claims = Claims::with_email(email);
+pub fn create_token(
+    user: &SlimUser,
+    issuer: String,
+    auth_duration_in_hour: u16,
+) -> Result<String, ServiceError> {
+    let claims: Claims = Claims::new(user, issuer, auth_duration_in_hour);
+
     encode(
         &Header::new(Algorithm::RS256),
         &claims,
-        include_bytes!("../../keys/private_rsa_key.der"),
+        include_bytes!("../../keys/rs256-4096-private.rsa"),
```

Comment:
Was recently no more compiling with the DER on my dev env...
Moreover, using the DER makes one more step on the `openssl` stuff, as `jsonwebtoken` allow us to directly use the pem format, let go for it…